src/components/navigation/ListItemLink.tsx

```diff
@@ -24,39 +23,12 @@ const ListItemLink = ({
     icon,
     title,
     link,
-    disabled,
     isOpen,
     menuWidth,
     badgeContent,
 }: Props) => {
     const intl = useIntl();
 
-    const RouterLink = forwardRef<HTMLAnchorElement, Omit<NavLinkProps, 'to'>>(
```

Comment:
Linting issue caught this known not great pattern... fixed it by breaking it out into a component.